### PR TITLE
CSS Modules README

### PR DESCRIPTION
**Pull request by mxstbr**
Based on the format of the Aphrodite README.

Please review contents @geelen and merge if you think it's alright

### CODE REVIEW

File: examples/css-modules/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- ✅ Small and lightweight
+- ✅ Supports global CSS
+- ✅ Supports entirety of CSS
+- 😕 Colocated (_has_ to be in a separate file)
+- ✅ Isolated
+- ✅ Doesn’t break inline styles
+- ✅ Easy to override
+- 😕 Theming (it's CSS, so it works, but it's custom for each implementation, i.e. no standard way of doing it)
+- ✅ Pre-build
+- ❌ Server side rendering
```
Comment (geelen) [inline examples/css-modules/README.md]: I... think CSS Modules can do this, but I've never tried.

**Review by geelen — APPROVED** [review]

File: examples/css-modules/README.md (hunk 1 above)
Comment (mxstbr) [reply examples/css-modules/README.md]: IIRC one of the big reasons @JedWatson switched to Aphrodite was that it had (better maybe?) SSR support, which CSS Modules doesn't?

Comment (geelen) [reply examples/css-modules/README.md]: I think it's mainly that running your server through webpack to understand `require('./styles.css')` is a real pain. Dunno tho.

Comment (MicheleBertoli) [reply examples/css-modules/README.md]: @mxstbr CSS Modules work with SSR, running the app through Webpack (as @geelen said) and it's not that painful :)
I've done it once, even if I usually extract the CSS at build time.

**Pull request merged**